packages/hardhat/contracts/Ethernauts.sol

```diff
@@ -173,7 +199,9 @@ contract Ethernauts is ERC721Enumerable, Ownable {
     // -----------------------
 
     function gift(address to) external onlyOwner {
-        require(_tokensGifted < maxGiftable, "No more Ethernauts can be gifted");
+        if (_tokensGifted > maxGiftable) {
```

Comment:
This should be `>=` instead of `>` otherwise if `_tokensGifted == maxGiftable` then an error won't be thrown